e2e_test.go

```diff
@@ -133,9 +134,11 @@ func TestE2E(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
 	b, err := ioutil.ReadAll(r)
 	if err != nil {
 		t.Fatal(err)
 	}
+
 	fmt.Println(string(b))
```

Comment:
Is this a debug print?

Reply:
I think this was there before. The e2e test seems to just do the query and print it out.
I just added the whitespace as I have a thing about gutters after close curly brace 😂

Reply:
I think a bigger piece needs to come next to do some real assertions on the returned data. I will leave this there until then.